staging/src/k8s.io/client-go/metadata/metadatainformer/interface.go

```diff
@@ -20,12 +20,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/cache"
 )
 
 // SharedInformerFactory provides access to a shared informer and lister for dynamic client
 type SharedInformerFactory interface {
 	Start(stopCh <-chan struct{})
 	ForResource(gvr schema.GroupVersionResource) informers.GenericInformer
+	ForExistingStoppableResource(gvr schema.GroupVersionResource) (informers.GenericInformer, cache.DoneChannel, bool)
```

Comment:
simmilar to pr #98657, this is breaking change to the interface. if people have concern with #98657, then this needs revisit too.

Reply:
Added a note to the agenda for the next sig-api-machinery meeting